### DISCUSSION TITLE
How to reset traceId by manual apis?

### DISCUSSION THREAD

**Discussion by mylwh** [apache/skywalking#10153] [Q&A] [answered]
We have multiple subsystems that communicate with each other via http. These subsystems all use skywalking and we don't want the trace from one subsystem to be passed to the downstream subsystem, so we would like to reset the trace bound to the upstream http request and create a new trace in the downstream subsystem by using a manual api. does skywalking support this now? If not, any suggestions on a good way to handle this?

**Comment by wu-sheng** [comment, marked as answer]
Then you don't need distributed tracing. This is not what SkyWalking was made for.

**Reply by wu-sheng** [reply]
There is no such APIs in all agents and SDKs AFAIK. Users would be confused if they saw this.

**Comment by Superskyyy** [comment]
It's really a weird use case, why intentionally breaking traces when its main purpose is to link things together?
